README.md

```diff
@@ -1,8 +1,7 @@
 Distro - an OS platform information API
 =======================================
 
-[![Build Status](https://travis-ci.org/nir0s/distro.svg?branch=master)](https://travis-ci.org/nir0s/distro)
-[![Build status](https://ci.appveyor.com/api/projects/status/e812qjk1gf0f74r5/branch/master?svg=true)](https://ci.appveyor.com/project/nir0s/distro/branch/master)
+[![Build Status](https://github.com/jdufresne/distro/workflows/Build/badge.svg)](https://github.com/jdufresne/distro/actions/workflows/build.yaml)
```

Comment:
We should probably use `github.com/python-distro/distro`?

Reply:
Thanks for catching this. I had intended to go back and fix after testing. Fixed now in latest.